packages/create-strapi-starter/utils/build-starter.js

```diff
@@ -0,0 +1,195 @@
+'use strict';
+
+const { resolve, join, basename } = require('path');
+const os = require('os');
+const fse = require('fs-extra');
+const fetch = require('node-fetch');
+const chalk = require('chalk');
+const tar = require('tar');
+
+const generateNewApp = require('strapi-generate-new');
+const GitUrlParse = require('git-url-parse');
```

Comment:
not a constructor so you can use camelCase `const parseGitUrl = require('git-url-parse')`